luminance/src/tess.rs

```diff
@@ -138,6 +140,11 @@ pub enum Mode {
   ///
   /// > This kind of primitive mode allows the usage of _primitive restart_.
   TriangleStrip,
+  /// A general purpose primitive with _n_ vertices, for use in tessellation shaders.
+  ///
+  /// If you want to employ tesselation shaders, this is the only primitive mode you can use. The
+  /// number of vertices in each primitive is set with [`TessBuilder::set_patch_vertex_nb`].
+  Patch,
```

Comment:
Hm, now I’m thinking about something. Why not remove the `patch_vert_nb` field, which will always be present even if one doesn’t do any tessellation, and put the value in this variant, like in `Patch(usize)`?

Reply:
This is even more true when you think about `Mode::Triangle`, which implicitly has 3 vertices, or `Mode::Point` which implies 1.

Reply:
Genius! Lots more idiomatic. Although I'm not sure I understand what you meant with `Triangle` and `Point`?

Reply:
Just some inner thoughts. :D They have _implicitly_ encoded the number of vertices (a triangle has 3 vertices), so it makes sense to put that value on `Patch`, since it can be arbitrary.

Reply:
Right!